### PR TITLE
cnf-tests: reporter: Grab MetalLB logs

### PR DESCRIPTION
**Pull request by sabinaaledort**
Adding MetalLB namespace for reporter to grab MetalLB logs as well.

### CODE REVIEW

File: cnf-tests/testsuites/pkg/utils/reporter.go [hunk 1]
```diff
@@ -92,6 +94,7 @@ func NewReporter(reportPath string) (*k8sreporter.KubernetesReporter, error) {
 		{Cr: &ocpbuildv1.BuildConfigList{}, Namespace: &namespaces.SroTestNamespace},
 		{Cr: &ocpbuildv1.BuildList{}, Namespace: &namespaces.SroTestNamespace},
 		{Cr: &netattdefv1.NetworkAttachmentDefinitionList{}},
+		{Cr: &metallbv1beta1.MetalLBList{}},
```
Comment (SchSeba) [inline cnf-tests/testsuites/pkg/utils/reporter.go:97]: you also need to add the scheme here https://github.com/openshift-kni/cnf-features-deploy/pull/1212/files#diff-1187a9bc7c5f7569a86f8677f5cdd3edba0b2eb90251d22208cbbec8de28f58dL40

File: hack/common.sh [hunk 2]
```diff
@@ -19,6 +19,9 @@ export OCP_VERSION="${OCP_VERSION:-4.11}"
 export OPERATOR_VERSION="${OPERATOR_VERSION:-4.11}"
 export GATEKEEPER_VERSION="${GATEKEEPER_VERSION:-v0.2.0}"
 
+# the metallb-operator deployment and test namespace
+export OO_INSTALL_NAMESPACE="${OO_INSTALL_NAMESPACE:-openshift-metallb-system}"
```
Comment (SchSeba) [inline hack/common.sh:23]: why we need this variable?

Comment (sabinaaledort) [reply hack/common.sh:23]: for the metallb-operator tests https://github.com/openshift/metallb-operator/blob/a2f8cb1d9d96a3729c4dd4799aebf1dfc7751cf5/test/e2e/functional/tests/e2e.go#L31

**Pull request merged**